hustvl_yolop.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: YOLOP
+summary: YOLOP pretrained on the BDD100K dataset
+image: yolop.png
+author: Hust Visual Learning Team
+tags: [vision]
+github-link: https://github.com/hustvl/YOLOP
+github-id: hustvl/YOLOP
+featured_image_1: no-image
+featured_image_2: no-image
+accelerator: cuda-optional
+---
+## Before You Start
+Start from a **Python>=3.7** environment with **PyTorch>=1.7** and **torchvision>=0.8** installed. To install YOLOP dependencies:
+```bash
+conda install pytorch==1.7.0 torchvision==0.8.0 cudatoolkit=10.2 -c pytorchconda install pytorch==1.7.0 torchvision==0.8.0 cudatoolkit=10.2 -c pytorch
+```

```

Comment:
@Riser6 Do you think these are necessary?

According to the previous test runs, I think the model passes on pytorch latest version - which is preferable, so maybe we can remove these instructions?

Also regarding the previous netlify issue: it's fixed now, we just needed to merge with `master` lol. There's another CI issue that I'm investigating but it's unrelated to this PR :)

The PR is good to go on my side but I'm just wondering if we could remove the instructions above, LMK what you think

Reply:
OK, thank you very much, our model was trained with PyTorch =1.7 version, but the latest version should also be ok, because I think pyTorch has good forward compatibility.

Reply:
So I want to know when will this model be posted on your official website? Thanks!

Reply:
Before merging would you mind removing these lines then? thanks

Reply:
I have removed these lines,  looking forward to your merging! Thanks!